From 8ff6234dc3f9d3d83bc104b4b3c0515f48aa305a Mon Sep 17 00:00:00 2001
Message-Id: <8ff6234dc3f9d3d83bc104b4b3c0515f48aa305a@dist-git>
From: John Ferlan <jferlan@redhat.com>
Date: Tue, 16 Feb 2016 11:55:11 +0100
Subject: [PATCH] util: Fix virCgroupNewMachine ATTRIBUTE_NONNULL args

https://bugzilla.redhat.com/show_bug.cgi?id=1308494

Commit id 'c3bd0019c0' removed arg3, but forgot to adjust the numbers
for NONNULL - caused build failure for coverity

(cherry picked from commit b8c0f186543c01cbda95c7251a0a992823764df5)
Signed-off-by: Martin Kletzander <mkletzan@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/util/vircgroup.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/util/vircgroup.h b/src/util/vircgroup.h
index 6570cb3..b98e416 100644
--- a/src/util/vircgroup.h
+++ b/src/util/vircgroup.h
@@ -110,7 +110,7 @@ int virCgroupNewMachine(const char *name,
                         int controllers,
                         virCgroupPtr *group)
     ATTRIBUTE_NONNULL(1) ATTRIBUTE_NONNULL(2)
-    ATTRIBUTE_NONNULL(4);
+    ATTRIBUTE_NONNULL(3);
 
 int virCgroupTerminateMachine(const char *name)
     ATTRIBUTE_NONNULL(1);
-- 
2.7.2

